src/main/java/com/bugsnag/android/ErrorStore.java

```diff
@@ -76,7 +76,9 @@ void write(Error error) {
         String filename = String.format("%s%d.json", path, System.currentTimeMillis());
         try {
             Writer out = new FileWriter(filename);
```

Comment:
Unrelated, but `Writer out` should be outside the try/catch and a null check + close in `finally`